packages/shared/components/molecules/AliasDetails.svelte

```diff
@@ -0,0 +1,59 @@
+<script lang="typescript">
+    import { AddressBox, KeyValueBox } from 'shared/components/atoms'
+    import { Pill } from 'shared/components'
+    import { formatDate, localize } from '@core/i18n'
+    import { activeProfile } from '@core/profile'
+    import { formatTokenAmountPrecise } from '@core/wallet'
+    import { BASE_TOKEN } from '@core/network'
+
+    export let storageDeposit = 0
+    export let giftedStorageDeposit = 0
+    export let aliasId: string = null
+    export let governorAddress: string = null
+    export let stateControllerAddress: string = null
+    export let transactionTime: Date = null
+
+    $: formattedTransactionTime = formatDate(transactionTime, { dateStyle: 'long', timeStyle: 'medium' })
+    $: hasStorageDeposit = storageDeposit || (storageDeposit === 0 && giftedStorageDeposit === 0)
+
+    $: formattedStorageDeposit = formatTokenAmountPrecise(
+        storageDeposit ?? 0,
+        BASE_TOKEN[$activeProfile?.networkProtocol]
+    )
+
+    let detailsList: { [key in string]: { data: string; tooltipText?: string; isCopyable?: boolean } }
+    $: detailsList = {
+        transactionTime: { data: formattedTransactionTime },
+        ...(hasStorageDeposit && {
+            storageDeposit: {
+                data: formattedStorageDeposit,
+                tooltipText: localize('tooltips.transactionDetails.incomingstorageDeposit'),
+            },
+        }),
+        governorAddress: { data: governorAddress, isCopyable: true },
+        stateControllerAddress: { data: stateControllerAddress, isCopyable: true },
+    }
+</script>
+
+<transaction-details class="w-full h-full space-y-6 flex flex-auto flex-col flex-shrink-0">
+    <main-content class="flex flex-auto w-full flex-col items-center justify-center space-y-3">
+        <transaction-status class="flex flex-row w-full space-x-2 justify-center">
+            <Pill backgroundColor="green-300" darkBackgroundColor="gray-200">
+                {localize('pills.aliasCreated')}
+            </Pill>
```

Comment:
Does consider the pending state. I think this can be included in a general activity inclusion status pill. And then you can wrap that for the transactions with a transaction inclusion pill as that one has more props to determine the text displayed.

Reply:
i think for this 1 case it doesn't make much sense to create a new Pill component